easybuild/easyconfigs/m/MrBayes/MrBayes-3.2.6-gompi-2020b.eb

```diff
@@ -0,0 +1,26 @@
+name = 'MrBayes'
+version = '3.2.6'
+
+homepage = "https://nbisweden.github.io/MrBayes/"
+description = """MrBayes is a program for Bayesian inference and model choice across
+ a wide range of phylogenetic and evolutionary models."""
+
+toolchain = {'name': 'gompi', 'version': '2020b'}
+# add -fcommon as workaround for 'multiple definition' linking errors;
+# see also https://www.gnu.org/software/gcc/gcc-10/porting_to.html
+toolchainopts = {'usempi': True, 'extra_cflags': '-fcommon'}
+
+source_urls = ['https://github.com/NBISweden/MrBayes/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['f8fea43b5cb5e24a203a2bb233bfe9f6e7f77af48332f8df20085467cc61496d']
+
+builddependencies = [
+    ('Autotools', '20200321'),
+]
+
+dependencies = [
+    ('libreadline', '8.0'),
+    ('beagle-lib', '3.1.2'),
+]
+
```

Comment:
how about adding something like `sanity_check_commands = ['mb <<< %s' % x for x in ['about', 'help']]`?

Reply:
although I guess that should go in the easyblock

Reply:
Yeah, easyblock is a better place for that, exactly.

Reply:
https://github.com/easybuilders/easybuild-easyblocks/pull/2727